packages/plugin-core/src/commands/NoteLookupAutoCompleteCommand.ts

```diff
@@ -1,6 +1,10 @@
 import { DENDRON_COMMANDS } from "../constants";
 import { BasicCommand } from "./base";
-import { UI_NOTE_LOOKUP_COMMAND } from "./NoteLookupCommand";
+import {
+  AUTO_COMPLETABLE_COMMAND_ID,
+  UIAutoCompletableCmds,
+} from "../utils/autoCompleter";
+// import { AUTO_COMPLETABLE_COMMAND_IDS, getUIAutoCompletableUICmd } from "../utils/autoCompleter";
```

Comment:
nitpick: remove comment

Reply:
will merge into mainline but could you remove the above in a separate commit? (you can bundle it with your next PR)